digiroad2-api-oth/src/test/scala/fi/liikennevirasto/digiroad2/Digiroad2ApiSpec.scala

```diff
@@ -133,7 +133,7 @@ class Digiroad2ApiSpec extends AuthenticatedApiSpec with BeforeAndAfter {
   test("get enumerated property values", Tag("db")) {
     getWithUserAuth("/enumeratedPropertyValues/10") {
       status should equal(200)
-      parse(body).extract[List[EnumeratedPropertyValue]].size should be(14)
+      parse(body).extract[List[EnumeratedPropertyValue]].size should be(13)
```

Comment:
The query in "queries.scala" file was wronged, returns all type_id, not only the type_id = 10. When we include the new ones we saw that the query improperly returns 16.. We correct the query and the test above.